fireworks/flask_site/app.py

```diff
@@ -407,4 +406,16 @@ def simple(coll, interval, num_intervals):
 
 
 if __name__ == "__main__":
-    app.run(debug=True, port=8080)
+    # https://dlukes.github.io/flask-wsgi-url-prefix.html
+
+    from fireworks import LaunchPad
+    from werkzeug.middleware.dispatcher import DispatcherMiddleware
```

Comment:
Is this a new dependency or something that is already required by Flask?

Reply:
Flask has been built on top of werkzeug so it is already required.